portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/pages/recovery/forced-password-reset.hbs

```diff
@@ -0,0 +1,44 @@
+{{secured}}
+{{layout "privilege"}}
+{{favicon "images/favicon.png" type="image/png"}}
+{{title "Reset Password | " @config.appName}}
```

Comment:
We can internationalize the tiltle. As there can be need to use other languages.
You can do it by 
{{title (i18n "users.add.header") " | " @config.appName}}